### PR TITLE
Add docs for creating a new specialist document type

### PR DESCRIPTION
**Pull request by tijmenb**
First version. Can be merged now but will be expanded in the future.

@Davidslv @klssmith

### CODE REVIEW

File: docs/creating-a-new-specialist-document-type.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+You'll have to make sure that when this app sends new documents to the
+publishing-api it can validate your request against the
+[govuk-content-schemas][govuk-content-schemas].
+
+## 2. Configure specialist-publisher
+
+You could use the [PR to create the International Development Fund](https://github.com/alphagov/specialist-publisher/pull/855) document as a
+template.
+
+The two most important things here are the document class, like [`AaibReport`](app/models/aaib_report.rb), and the finder configuration file, like [aaib_reports.json](lib/documents/schemas/aaib_reports.json).
```
Comment (klssmith) [inline docs/creating-a-new-specialist-document-type.md]: The 2 links in this paragraph don't seem to be working

File: docs/creating-a-new-specialist-document-type.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## 3. Configure rummager
+
+You'll need to add the document type to search (the [rummager][rummager] application), so that it can
+index the new documents.
```
Comment (klssmith) [inline docs/creating-a-new-specialist-document-type.md:17]: Would it be worth including where in Rummager to add the document type? (We had trouble trying to find where to add it earlier.)

File: docs/creating-a-new-specialist-document-type.md (hunk 1 above)
Comment (tijmenb) [reply docs/creating-a-new-specialist-document-type.md]: Updated.

File: docs/creating-a-new-specialist-document-type.md (hunk 2 above)
Comment (tijmenb) [reply docs/creating-a-new-specialist-document-type.md:17]: I'll add some docs to rummager first, and then add the link here. Sounds good?

Comment (klssmith) [reply docs/creating-a-new-specialist-document-type.md:17]: Yes, that would be great.

**Pull request merged**